tests/test_package.py

```diff
@@ -44,8 +44,8 @@ def test_require_no_url_req():
 @pytest.mark.parametrize("hw_extra", sorted(HW_EXTRAS))
 @pytest.mark.parametrize("py_version", ["3.10", "3.11"])
 def test_package_conflict(py_version: str, hw_extra: str) -> None:
-    if py_version == "3.11" and hw_extra == "hpu":
-        pytest.skip("Intel Gaudi is not supported on 3.11")
+    if py_version == "3.10" and hw_extra == "hpu":
```

Comment:
should this be `py_version != "3.11"` instead?

Reply:
Makes sense, fixed.